app/helpers/preferences_helper.rb

```diff
@@ -2,18 +2,16 @@ module PreferencesHelper
 
   def pref(model, pref_name, &block)
     s = content_tag(:label, Preference.human_attribute_name(pref_name), :for => model+pref_name)
-    s << "<br/>".html_safe
     s << yield
-    s << "<br/><br/>".html_safe
     s
   end
 
   def pref_with_select_field(model, pref_name, collection = [ [t('preferences.is_true'),true], [t('preferences.is_false'), false] ])
-    pref(model, pref_name) { select(model, pref_name, collection) }
+    pref(model, pref_name) { select(model, pref_name, collection, {}, class: "form-control") }
```

Comment:
i prefer double quotes because it's one less thing that has to change should interpolation be needed (and it's not any slower than single quotes)